projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -34,6 +34,13 @@ abstract class Base_Admin_Menu {
 	 */
 	protected $domain;
 
+	/**
+	 * The CSS classes used to hide the submenu items in navigation.
+	 *
+	 * @var string
+	 */
+	const HIDE_CSS_CLASS = 'hide-if-js hide-if-no-js';
```

Comment:
Can we also consider other hiding classes like `hidden`?  See https://core.trac.wordpress.org/browser/trunk/src/wp-admin/css/common.css?rev=50571#L166.

Reply:
I thought about that as well, but I wanted to piggyback on the existing filtering in [WPCOM API](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php#L223).

We could look into it, but I'm not sure what the benefits would be. :)

Reply:
In that case, I'd remove then the `hide-if-no-js` class here, so we're aligned with the API endpoint